src/client/power-pages/telemetry.ts

```diff
@@ -59,8 +63,8 @@ export function sendTelemetryEvent(telemetry: ITelemetry, telemetryData: IPowerP
 
 export function sendTelemetryEventWithTriggerPoint(entityType: Tables, uri:vscode.Uri, telemetry:ITelemetry): void {
```

Comment:
@amitjoshi438 Do we really need this extra function in telemetry? Why not use a helper function like uri ? TriggerPoint1: TriggerPoint2 and send that directly in the telemetrydataProperties using the original telemetry function?

Reply:
right, we can avoid that extra function.